tests/sentry/models/test_exporteddata.py

```diff
@@ -120,3 +121,22 @@ def test_email_success_content(self, builder):
             "html_template": "sentry/emails/data-export-success.html",
         }
         assert builder.call_args[1] == expected_email_args
+
+    def test_email_failure(self):
+        with self.tasks():
+            self.data_export.email_failure(self.TEST_STRING)
+        assert len(mail.outbox) == 1
+        assert not ExportedData.objects.filter(id=self.data_export.id).exists()
+
+    @patch("sentry.utils.email.MessageBuilder")
+    def test_email_failure_content(self, builder):
+        with self.tasks():
+            self.data_export.email_failure(self.TEST_STRING)
+        expected_email_args = {
+            "subject": "Unable to Export Data",
+            "context": {"error_message": self.TEST_STRING},
+            "type": "organization.export-data",
+            "template": "sentry/emails/data-export-failure.txt",
+            "html_template": "sentry/emails/data-export-failure.html",
+        }
+        assert builder.call_args[1] == expected_email_args
```

Comment:
@leeandher I think you should use `assert_called_with` instead of this kind of assertion if possible

Reply:
Done!